lib/samples/show_device_location_with_nmea_data_sources/README.md

```diff
@@ -0,0 +1,56 @@
+# Show device location with NMEA data sources
+
+Parse NMEA sentences and use the results to show device location on the map.
+
+![Image of show device location with NMEA data sources](show_device_location_with_nmea_data_sources.png)
+
+## Use case
+
+NMEA sentences can be retrieved from GPS receivers and parsed into a series of coordinates with additional information. Devices without a built-in GPS receiver can retrieve NMEA sentences by using a separate GPS dongle, commonly connected bluetooth or through a serial port.
```

Comment:
I feel like there's a missing word in front of "bluetooth" (which should be capitalized too). "commonly connected via Bluetooth"